### ISSUE TITLE
No support for 5-star-rating as a column

### ISSUE THREAD

**Issue by exaveal** [lotan/rhythmbox-ampache#26]
When connecting to an ampache music server, rhythmbox does not show the 5-star-rating of the music files in a column.
Can you please fix this?

**Comment by lotan** [comment]
I can confirm that the ratings do not seem to work for Ampache.

You can see here
https://github.com/lotan/rhythmbox-ampache/blob/master/AmpacheBrowser.py#L140
that the ratings are registered with Rhythmbox for each song.

However, the ratings column is only shown on the main music database, not for the Ampache plugin.

I imagine the idea of the Rhythmbox team is that you can only rate the Rhythmbox local database. But I can only guess.

**Comment by exaveal** [comment]
Yes, but why not take over full functionality for ampache? Meaning: Read the Rating of a song from Ampache and show it in the column "Rating". When changing the rating in Rhythmbox, write it back to Ampache.
With this working, Ampache can be used via native desktop  client Rhythmbox + this plugin
